Tests/SwiftDocCUtilitiesTests/OutOfProcessReferenceResolverTests.swift

```diff
@@ -371,32 +371,32 @@ class OutOfProcessReferenceResolverTests: XCTestCase {
     func testForwardsErrorOutputProcess() throws {
```

Comment:
Recently I've just been prefixing the test method name with `skip_` to resolve this issue. Requires less code change and doesn't cause an XCTSkip warning.

```swift
func testForwardsErrorOutputProcess() throws {
```

to

```swift
// This test is flaky rdar://88498898
func skip_testForwardsErrorOutputProcess() throws {
```